conanfile.py

```diff
@@ -159,6 +159,7 @@ def generate(self):
         tool_ch.variables['USERVER_IS_THE_ROOT_PROJECT'] = False
         tool_ch.variables['USERVER_DOWNLOAD_PACKAGES'] = True
         tool_ch.variables['USERVER_PIP_USE_SYSTEM_PACKAGES'] = True
+        tool_ch.variables['USERVER_PIP_OPTIONS'] = ""
```

Comment:
This is not the way to keep the default value for an option
Just remove the line